### PR TITLE
Bind_event_listener_when_registering_display_object

### PR DESCRIPTION
**Pull request by sudhir-jedhe**

### CODE REVIEW

File: src/widgets/CheckBox.js
Comment (mnewcomb314) [inline src/widgets/CheckBox.js]: Nit: please delete stale code

File: src/widgets/CheckBox.js
Comment (mnewcomb314) [inline src/widgets/CheckBox.js]: Nit: please delete stale code

File: src/widgets/Button.js
Comment (mnewcomb314) [inline src/widgets/Button.js:20]: I like this better than what I was thinking of for event registration!  And it has the added flexibility of allowing for binding multiple listeners for the same event.

**Comment by mnewcomb314** [comment]
Looks good!  Just some minor nits, and I'm also holding this PR until the CAM `1.0.0-prerelease.3` is done which will be after your corresponding CAM PR merges (and once that build exists, I will ask you to please update package*.json to point to that version of CAM).

File: src/widgets/CheckBox.js
Comment (sudhir-jedhe) [reply src/widgets/CheckBox.js]: removed

File: src/widgets/CheckBox.js
Comment (sudhir-jedhe) [reply src/widgets/CheckBox.js]: removed

**Comment by sudhir-jedhe** [comment]
@mnewcomb314  updated CAM version

**Review by mnewcomb314 — APPROVED** [review]

**Pull request merged**
